### PR TITLE
Internally separate error from result in get API results

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Internally separate error from result in get API results
- Add enableErrorResultSeparationForGetApis dev flag

Note:
- With enableErrorResultSeparationForGetApis = false, the APIs are backward-compatible.

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/1053

### CODE REVIEW

File: db/index.js [hunk 1]
```diff
@@ -525,18 +538,34 @@ class DB {
   }
 
   getValue(valuePath, options) {
+    return this.getValueWithError(valuePath, options).result;
```
Comment (minsulee2) [inline db/index.js:541]: Is it guaranteed it always have result?

**Review by minsulee2 — APPROVED** [review]
LGTM with minor question!

File: db/index.js (hunk 1 above)
Comment (platfowner) [reply db/index.js:541]: Yes, we can see that readFromStateRoot() always returns 'result'.

**Comment by platfowner** [comment]
Thanks for the fast review! Now merging.. @minsulee2

**Pull request merged**